packages/core/src/StellarCore.js

```diff
@@ -27,8 +23,12 @@ class StellarCore {
     }
 
     this.source = source;
-    this.sourceResolver(source);
-    this.log.info(`@StellarCore.setSource`, { source });
+    this.setMiddlewares();
```

Comment:
Maybe I missed something, but I think `setMiddlewares` is not defined here (in StellarCore class)

Reply:
yeah it isn't - its implemented in the child classes. i'll add method that throws an error